### DISCUSSION TITLE
NEED ANWSERS BEACUSE I THINK NO ONE HAS EVER SEEN THIS!

### DISCUSSION THREAD

**Discussion by glowsoony** [KadeArchive/Kade-Engine#2692] [Q&A] [answered]
I ADDED MONOCHROME TO KADE ENGINE 1.6.2 AND NO THERE NO CUSTOM NOTES BUT I WENT INTO CHART BECAUSE I THOUGHT MY MIND WAS MESSING WITH ME AS I SAW A GRAY NOTE! NO CUSTOM NOTS I NEVER EVEN COULD ADD THEM AND A GRAY NOTE? SOMEONE PLEASE EXPLAIN AND SOMETHING IS ODD AS THIS ONLY HAPPENS WHEN bf_dark IS OPPENET. glowsoony as note style is not custom notes its just note_assets replaced so no its not gray notes if yall ask.
![image](https://user-images.githubusercontent.com/84847356/140652622-a3d481fc-e59a-4f02-8a84-a00832f69159.png)
![image](https://user-images.githubusercontent.com/84847356/140652668-2d22dbd7-5216-4856-b483-e06f8bd3fb81.png)
![image](https://user-images.githubusercontent.com/84847356/140652816-cdb9df42-4d75-4224-92fe-d319daf6b8fe.png)

**Comment by Jotaro-Gaming** [comment, marked as answer]
this is a glitch that rarely happens 
what it did was use the normal down asset as a note you can safely remove it

**Comment by libracrumbs** [comment]
there is a reply  here explaining it but just asking cuz dont know, was hypnos lullaby made in pe? it reminds me of where the event arrows go

**Reply by glowsoony** [reply]
It might be the event arrows just bugged
